### PR TITLE
Sending events and logs to Datadog with Amazon EventBridge API Destinations

### PR DESCRIPTION
**Pull request by apigirl**
## What does this PR do?
Adds a guide around the new(ish) Datadog support in Eventbridge destinations

https://docs-staging.datadoghq.com/kaylyn/eventbridge/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations/

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by jimmycaputo** [comment]
Step 3 is a little confusing. This is only relevant if your rule is triggering when data is uploaded to S3, which is the example in the AWS post, but S3 is only one of many services that post events to EventBridge. Might be better to just combine 3 and 4 and say you'll see your logs or events show up in Datadog a few minutes after your rules with Datadog destinations trigger?

Also, should we add a link in the intro to the other EventBridge integration page and directs readers there if looking to push events to EventBridge:
https://docs.datadoghq.com/integrations/amazon_event_bridge/#overview

I could see someone finding this page and being confused about how to push events. Or vice versa if they go to the EventBridge integration page and are looking to send events to Datadog.

**Comment by apigirl** [comment]
@jimmycaputo - thanks for the feedback! I've updated based on it. I did not remove the S3 example entirely, because I still think it's a really good example, but I did make it more clear that it was one way of many, with your suggested more clear links to how to send events from Datadog to EventBridge. I am also going to open up a separate PR to add a link from the EventBridge integration docs to this guide.

**Review by jimmycaputo — APPROVED** [review]
LGTM

File: content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+Amazon EventBridge is a serverless event bus that enables you to build event-driven applications. EventBridge can integrate with your AWS services, but the API destinations feature lets you push and pull data from outside of AWS using APIs. This guide gives steps for sending your events and logs from EventBridge to Datadog. For more information about pushing your events from Datadog to EventBridge, [see the EventBridge integration docs][1].
+
+## Setup
+
+Before you begin, you need a [Datadog account][2], with [an API key][3], and you need access to [Amazon Eventbridge API destinations][4].
+
+### Configuration
+
+1. Follow the steps in the [Amazon Create an API destination docs][5] to add Datadog as an API destination.
+    - Use API key Authorization, with `DD-API-KEY` as your key name and your [Datadog API key][3] as the value.
```
Comment (jtappa) [inline content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md:19]:
```suggestion
    - Use API key authorization, with `DD-API-KEY` as your key name and your [Datadog API key][3] as the value.
```

File: content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - For your destination endpoint, input `https://http-intake.logs.datadoghq.com/v1/input` for logs or `https://api.datadoghq.com/api/v1/events` for events, and set `POST` as the HTTP method. For more information about the differences between logs and events, see the [logs section][6], and the [events section][7] of the [Categories of Data docs page][8].
```
Comment (jtappa) [inline content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md:20]:
```suggestion
    - For your destination endpoint, use `https://http-intake.logs.datadoghq.com/v1/input` for logs or `https://api.datadoghq.com/api/v1/events` for events, and set `POST` as the HTTP method. For more information about the differences between logs and events, see the [logs section][6], and the [events section][7] of the [Categories of Data docs page][8].
```

File: content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+2. Once you've set up the destination, you can now follow the Amazon instructions to [create an EventBridge rule][9], where you set Datadog as your destination.
+3. Once you've set up the rule with Datadog as the destination, trigger an event by posting an event to EventBridge. For more information about pushing events to EventBridge from Datadog, see the [EventBridge integration docs][1]. For example, to trigger a test event by [uploading the objects to an S3 bucket][10] in your account, use this AWS CloudShell cmd:
```
Comment (jtappa) [inline content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md:22]:
```suggestion
3. Once you've set up the rule with Datadog as the destination, trigger an event by posting an event to EventBridge. For more information about pushing events to EventBridge from Datadog, see the [EventBridge integration docs][1]. For example, to trigger a test event by [uploading the objects to an S3 bucket][10] in your account, use this AWS CloudShell command:
```

File: content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+    ```bash
+    echo "test" > testfile.txt
+    aws s3 cp testfile.txt s3://YOUR_BUCKET_NAME
+    ```
+4. Once events and logs are sending, after about five minutes, you start seeing the data in the Datadog [logs console][11] or [events stream][12], depending on which endpoint you are sending them.
```
Comment (jtappa) [inline content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md:28]:
```suggestion
4. Once events and logs are sending, after about five minutes, the data is available in the Datadog [logs console][11] or [events stream][12], depending on which endpoint you are sending them to.
```

**Review by jtappa — CHANGES_REQUESTED** [review]

**Review by jtappa — APPROVED** [review]

**Pull request merged**